2021-01-19 08:22:31,255 INFO    MainThread:23855 [wandb_setup.py:_flush():70] setting env: {}
2021-01-19 08:22:31,255 INFO    MainThread:23855 [wandb_setup.py:_flush():70] multiprocessing start_methods=fork,spawn,forkserver
2021-01-19 08:22:31,256 INFO    MainThread:23855 [wandb_setup.py:_flush():70] setting login settings: {}
2021-01-19 08:22:31,256 INFO    MainThread:23855 [wandb_init.py:_log_setup():303] Logging user logs to /home/fkubota/Git/sand-pit/wandb/run-20210119_082231-2484buao/logs/debug.log
2021-01-19 08:22:31,256 INFO    MainThread:23855 [wandb_init.py:_log_setup():304] Logging internal logs to /home/fkubota/Git/sand-pit/wandb/run-20210119_082231-2484buao/logs/debug-internal.log
2021-01-19 08:22:32,254 INFO    MainThread:23855 [wandb_run.py:_console_start():1237] atexit reg
2021-01-19 08:22:32,255 INFO    MainThread:23855 [wandb_run.py:_redirect():1104] redirect: SettingsConsole.REDIRECT
2021-01-19 08:22:32,256 INFO    MainThread:23855 [wandb_run.py:_redirect():1107] Redirecting console.
2021-01-19 08:22:32,256 INFO    MainThread:23855 [redirect.py:install():204] install start
2021-01-19 08:22:32,257 INFO    MainThread:23855 [redirect.py:install():219] install stop
2021-01-19 08:22:32,257 INFO    MainThread:23855 [redirect.py:install():204] install start
2021-01-19 08:22:32,258 INFO    MainThread:23855 [redirect.py:install():219] install stop
2021-01-19 08:22:32,258 INFO    MainThread:23855 [wandb_run.py:_redirect():1151] Redirects installed.
2021-01-19 08:22:32,258 INFO    MainThread:23855 [wandb_config.py:__setitem__():135] config set i_fold = 0 - <bound method Run._config_callback of <wandb.sdk.wandb_run.Run object at 0x7f617e202160>>
2021-01-19 08:22:32,258 INFO    MainThread:23855 [wandb_run.py:_config_callback():549] config_cb i_fold 0 {'_wandb': {'cli_version': '0.10.12', 'python_version': '3.6.9', 'is_jupyter_run': False, 'is_kaggle_kernel': False}, 'i_fold': 0}
2021-01-19 08:22:32,260 INFO    MainThread:23855 [wandb_run.py:_atexit_cleanup():1207] got exitcode: 1
2021-01-19 08:22:32,261 INFO    MainThread:23855 [wandb_run.py:_restore():1179] restore
2021-01-19 08:22:32,261 INFO    MainThread:23855 [redirect.py:uninstall():223] uninstall start
2021-01-19 08:22:32,261 INFO    MainThread:23855 [redirect.py:_stop():278] _stop: stdout
2021-01-19 08:22:32,261 INFO    MainThread:23855 [redirect.py:_stop():284] _stop closed: stdout
2021-01-19 08:22:32,261 INFO    stdout    :23855 [redirect.py:_pipe_relay():120] relay done saw last write: stdout
2021-01-19 08:22:32,262 INFO    stdout    :23855 [redirect.py:_pipe_relay():136] relay done done: stdout
2021-01-19 08:22:32,262 INFO    MainThread:23855 [redirect.py:_stop():290] _stop joined: stdout
2021-01-19 08:22:32,262 INFO    MainThread:23855 [redirect.py:_stop():292] _stop rd closed: stdout
2021-01-19 08:22:32,262 INFO    MainThread:23855 [redirect.py:uninstall():227] uninstall done
2021-01-19 08:22:32,262 INFO    MainThread:23855 [redirect.py:uninstall():223] uninstall start
2021-01-19 08:22:32,262 INFO    MainThread:23855 [redirect.py:_stop():278] _stop: stderr
2021-01-19 08:22:32,262 INFO    MainThread:23855 [redirect.py:_stop():284] _stop closed: stderr
2021-01-19 08:22:32,262 INFO    stderr    :23855 [redirect.py:_pipe_relay():120] relay done saw last write: stderr
2021-01-19 08:22:32,262 INFO    stderr    :23855 [redirect.py:_pipe_relay():136] relay done done: stderr
2021-01-19 08:22:32,262 INFO    MainThread:23855 [redirect.py:_stop():290] _stop joined: stderr
2021-01-19 08:22:32,262 INFO    MainThread:23855 [redirect.py:_stop():292] _stop rd closed: stderr
2021-01-19 08:22:32,262 INFO    MainThread:23855 [redirect.py:uninstall():227] uninstall done
2021-01-19 08:22:32,766 INFO    MainThread:23855 [wandb_run.py:_wait_for_finish():1324] got exit ret: uuid: "093d2972ca2b4f2a9f82e993b542025b"
response {
  poll_exit_response {
    file_counts {
      wandb_count: 1
    }
    pusher_stats {
      uploaded_bytes: 711
      total_bytes: 711
    }
  }
}

2021-01-19 08:22:34,774 INFO    MainThread:23855 [wandb_run.py:_wait_for_finish():1324] got exit ret: uuid: "43a29e4c62404ebb92256876a51beb98"
response {
  poll_exit_response {
    done: true
    exit_result {
    }
    file_counts {
      wandb_count: 5
    }
    pusher_stats {
      uploaded_bytes: 4891
      total_bytes: 4891
    }
  }
}

2021-01-19 08:22:35,878 INFO    MainThread:23855 [wandb_run.py:_show_files():1536] logging synced files
2021-01-19 08:22:35,880 INFO    MainThread:23855 [internal.py:handle_exit():138] Internal process exited
